lib/isrlib/Config.uk

```diff
@@ -0,0 +1,4 @@
+config LIBISRLIB
+    bool "isrlib: Interrupt-safe standard routines"
+    default n
+    select LIBNOLIBC if !HAVE_LIBC
```

Comment:
should we select libc? If we need it for headers, should we move header into the library?

Reply:
We depend on standard libc-provided headers, especially for the data types like provided with `<stddef.h>`. Longer-term the prototypes should not differ to much from standard libc headers and we should be able to re-use things like `uint8_t`, `NULL`, etc.

Reply:
Yeah it make sense to use those header.

The other discussion on including the libc, need more support from the build system to prevent call from this library to the function that aren't to be called from a interrupt routine. As this extends the scope of this patch, we should treat it separate set.